unittests/DefToYAMLConverter/DefToYAMLConverterTest.cpp

```diff
@@ -65,7 +65,8 @@ TEST(DefToYAMLConverterTest, matchDiagnosticMessagesSequentially) {
   YAMLLocalizationProducer yaml(EnglishLocalization.str());
 
   yaml.forEachAvailable([](swift::DiagID id, llvm::StringRef translation) {
-    ASSERT_TRUE(diagnosticMessages[static_cast<uint32_t>(id)] == translation);
+    llvm::StringRef msg = diagnosticMessages[static_cast<uint32_t>(id)];
+    ASSERT_EQ(msg, translation);
```

Comment:
I had to store the `msg` in a variable so if `ASSERT_EQ` fails it will show a useful error message like: 
```
      Expected: msg
      Which is: "%0 now requires %select{|no}1 new witness table entry"
To be equal to: translation
      Which is: "something else..."
```